### DISCUSSION TITLE
Cannot add a foreign key to a new table

### DISCUSSION THREAD

**Discussion by lucian0c** [dbeaver/dbeaver#15934] [General]
- Ubuntu 20.04
- Version 21.3.5.202202200327

- MySQL 5.7
- connector-j/8.0

Creating two new tables, if the second has to be related to the first by a foreign key, the table does not appear in the list.

1) Create New table A with a field called id
2) Create a second table with field called aTableID
3) Try to add a foreign Key: the table A does not appear in the list

tried to refresh connection without any luck. Has to restart dbeaver

**Comment by LonwoLonwo** [comment]
Hello @lucian0c 

Unfortunately, I can't reproduce your issue.

![2022-02-28 10_34_54-Create foreign key _ Create foreign key for table _newtable_1_](https://user-images.githubusercontent.com/45152336/155942586-2ddea849-9205-4564-ad6c-fb88ec138401.png)

Please provide more info.

1. How were the tables created? Via DBeaver UI tools (like "Create table" in the Database Navigator tree) or manually in SQL Editor?
2. Have you tried to refresh the Tables list or Database in the Database Navigator tree? After table creation.
3. Maybe you have any error logs? You can read more about DBeaver logs here: https://github.com/dbeaver/dbeaver/wiki/Log-files
